windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -23,13 +23,34 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-5-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.1.0
+
+This release of the WMCO is now available with a bug fix and an improvement. The components of the WMCO 5.1.0 were released in link:https://access.redhat.com/errata/RHBA-2022:95425-01[RHBA-2022:95425-01].
+
+=== Bug fix
+
+Previously, the reverse DNS lookup of Windows Bring-Your-Own-Host (BYOH) instances failed when the node's external IP was present without pointer records (PTR). With this release, if the PTR record is not present in the first node IP address,
+the WMCO looks in the other node addresses until a reverse lookup record is found. As a result, the reverse configuration of Windows BYOH instances succeed when node external IP address is present without a PTR record.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2081825[**BZ#2081825**])
+
+=== Known Issue
+
+Windows machine sets can't scale up when the `publicIP machineset` field is set to `False` on Microsoft Azure. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091642[**BZ#2091642**])
```

Comment:
s/can't/cannot

Reply:
s/the `publicIP machineset` field is set to `False/the `publicIP` parameter is set to false in the machine set ??
@jfrancoa @jrvaldes @mtnbikenc

Reply:
@Amrita42 change suggested by Michael looks good to me, the wording should be:
the publicIP` parameter is set to false in the machine set